gamemodes/terrortown/gamemode/client/cl_targetid.lua

```diff
@@ -58,7 +58,7 @@ local materialBaseOverlay = Material("vgui/ttt/dynamic/sprite_base_overlay")
 -- @hook
 -- @realm client
 function GM:GetClassHints()
-	return ClassHint
+	return ClassHint -- this is most definitely not localized
```

Comment:
i have literally no idea what this would like to reference.

@TimGoll do you have an idea?

Reply:
I recently read a report (on discord I think) that hints aren't working anymore in TTT2 targetID. This might be the reason why it is broken. I propably removed this unintenionally:

https://github.com/Facepunch/garrysmod/blob/e189f14c088298ca800136fcfcfaf5d8535b6648/garrysmod/gamemodes/terrortown/gamemode/cl_targetid.lua#L11-L23

Reply:
Howver, looking at it I think there is more missing. Might have to be reworked a bit to fix it.

Reply:
i can't find any other reference to `ClassHint` in the original ttt code. Anyways i'll just throw it in again then we can see how much of it works again.

Reply:
I think there is more missing to make it work, because right now it is used nowhere

Reply:
`ClassHint` was removed in this commit 9cef0ced0edb102255a5377db5c610f2c2364d4f

so it's definitely not the recent reason for TargetID not working.

For this function we should probably return something else entirely, i have not found out what.

Reply:
It might be broken since then without anyone noticing though